actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -43,6 +43,18 @@ def initialize(params) # :nodoc:
     end
   end
 
+  # Raised when a Parameters instance is not marked as permitted and
+  # an operation to transform to hash is called.
+  #
+  #   params = ActionController::Parameters.new(a: "123", b: "456")
+  #   params.to_h
+  #   # => ActionController::UnfilteredParameters: unable to convert unpermitted parameters to hash
+  class UnfilteredParameters < StandardError
+    def initialize
+      super("unable to convert unpermitted parameters to hash")
```

Comment:
+1 exceptions should clearly state what's wrong, but leave speculation about how the developer might fix it to someone/something else